quic/s2n-quic-core/src/connection/limits.rs

```diff
@@ -120,11 +124,45 @@ impl Limits {
         unidirectional_data_window,
         u64
     );
-    setter!(
-        with_max_open_bidirectional_streams,
-        max_open_bidirectional_streams,
-        u64
-    );
+
+    /// Sets the max local and remote limits for bidi streams.
+    ///
+    /// The value `with_max_open_remote_bidirectional_streams` and
+    /// `with_max_open_local_bidirectional_streams` will be used instead
+    /// if set on the builder.
+    // TODO: Deprecate
+    pub fn with_max_open_bidirectional_streams(
+        mut self,
+        value: u64,
+    ) -> Result<Self, ValidationError> {
+        self.max_open_bidirectional_streams = value.try_into()?;
```

Comment:
Can we get remove this field now and have this just set `self.max_open_local_bidirectional_streams` and `self.max_open_remote_bidirectional_streams`?